src/projections/updateProjection.ts

```diff
@@ -8,10 +8,10 @@ import { debug, convertToCommandError } from "../utils";
 
 export interface UpdateProjectionOptions extends BaseOptions {
   /**
-   * Enables tracking emitted streams.
+   * Enables emitting events from the projection.
    * @default undefined
    */
-  trackEmittedStreams?: boolean | undefined;
+  emitEnabled?: boolean | undefined;
```

Comment:
**QUESTION:** Why do we have `| undefined` if it's optional? Shouldn't we just have?
```suggestion
  emitEnabled?: boolean;
```

Reply:
Yeah, probably, it means the same thing. 

I think my thinking back when I originally wrote that was that `undefined` is an acceptable value, and has meaning, and I wanted to imply that via the type.  e.g. "emit enabled can have the value of boolean or undefined. Its also optional to pass this."

Reply:
Removed, and updated the doc comment to explicitly say what undefined means.